test/validation/api/ipsec/ipsec.c

```diff
@@ -1092,32 +1097,23 @@ int ipsec_config(odp_instance_t ODP_UNUSED inst)
 
 int ipsec_term(odp_instance_t inst)
 {
-	odp_pool_t pool;
-	odp_queue_t out_queue;
-	odp_pktio_t pktio;
+	odp_pool_t pool = suite_context.pool;
+	odp_queue_t out_queue = suite_context.queue;
+	odp_pktio_t pktio = suite_context.pktio;
```

Comment:
It seems that depending on the test case some of these may be uninitilized. suite_context is static variable set to zero, but XXX_INVALID value is implementation defined (may not be zero). Maybe ipsec_init() should init all these handles to INVALIDs.

Reply:
Agreed. With V3, the suite_context fields pool, queue and pktio are initialized to corresponding XXX_INVALID values in ipsec_init().